build_tools/update_environments_and_lock_files.py

```diff
@@ -739,6 +767,10 @@ def main(select_build, skip_build, select_tag, verbose, very_verbose):
     filtered_conda_build_metadata_list = [
         each for each in filtered_build_metadata_list if each["type"] == "conda"
     ]
+    for each in filtered_build_metadata_list:
+        if isinstance(each["channel"], str):
```

Comment:
I guess use `channels` instead of `channel` and use a list in all the CI builds?

Reply:
Done